bin/i18n/resources/pegasus/emails/sync_out.rb

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env ruby
+
+require 'fileutils'
+
+require_relative '../../../i18n_script_utils'
+require_relative '../../../utils/pegasus_markdown'
+require_relative '../emails'
+
+module I18n
+  module Resources
+    module Pegasus
+      module Emails
+        class SyncOut < I18n::Utils::SyncOutBase
+          def process(language)
+            crowdin_file_path = I18nScriptUtils.locale_dir(language[:crowdin_name_s], DIR_NAME, FILE_NAME)
```

Comment:
When we download the translated content, the files will be in, for example:
```
i18n
 |
 +-- locales
      |
      +--Spanish
          |
          +--emails
              |
              +-- hoc_email_example.md
              +-- other_email_example.md
               ...
```
I don't think we currently have values assigned to `DIR_NAME` or `FILE_NAME` inside the Emails module.